app/data_model/answer_store.py

```diff
@@ -224,31 +174,7 @@ def upgrade_to_1_update_date_formats(answer_store, schema):
                 continue
 
 
-def upgrade_to_2_add_group_instance_id(answer_store, schema):
-    """ Answers should have a `group_instance_id` ready for more complex group repeat rules. """
-    from app.questionnaire.location import Location
-    from app.helpers.schema_helpers import get_group_instance_id
-
-    for answer in answer_store:
-        # Happily, parent_id's are patched on to the schema for each nested level, so we can
-        # retrieve the block_id and group_id for the answer we're processing here.
-        answer_schema = schema.get_answer(answer['answer_id'])
-        question = schema.get_question(answer_schema['parent_id'])
-        block_id = question['parent_id']
-        block = schema.get_block(question['parent_id'])
-        group_id = block['parent_id']
-
-        location = Location(
-            group_id=group_id,
-            group_instance=answer['group_instance'],
-            block_id=block_id,
-        )
-        # `get_group_instance_id` handles providing a consistent group_instance_id
-        answer['group_instance_id'] = get_group_instance_id(schema, answer_store, location, answer['answer_instance'])
-
-
 # Dictionary specifying upgrade methods. Key should be the version to upgrade to.
 UPGRADE_TRANSFORMS = {
     1: upgrade_to_1_update_date_formats,
-    2: upgrade_to_2_add_group_instance_id,
```

Comment:
I think we should remove upgrades entirely. We talked about rewriting the interface previously, and these upgrades aren't really relevant on v3.